jekyll/_cci2/server-3-install.adoc

```diff
@@ -581,6 +581,10 @@ and VM Service.
 CircleCI curates Terraform modules to help install Nomad clients in your cloud provider of choice. You can browse the modules
 in our https://github.com/CircleCI-Public/server-terraform[public repository].
 
+NOTE: We use `blocked_cidrs` to block network access to the specified CIDRs. This is a security feature to prevent Nomad Clients
+from communicating with services that should not be accessed by arbitrary code (such as spinning up VMs using the vm-service). Do
+not block access to/from VMs created by the VM Service as several features (such as building Docker containers) will cease to function.
```

Comment:
This is not what the variable does. The CIDR blocks in `blocked_cidrs` are taken to generate iptables rules to block access from within jobs only. Connectivity outside of the Docker realm isn't affected.